### PR TITLE
update openshift-apiserver-release-4.4 to use go-1.13

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
I think that `ci/rehearse/openshift/openshift-apiserve` fails because it requires https://github.com/openshift/openshift-apiserver/pull/104

```
STEP 5: RUN make build --warn-undefined-variables
build flag -mod=vendor only valid when using modules
make: Nothing to be done for `build'.
...
STEP 8: COPY --from=builder /go/src/github.com/openshift/openshift-apiserver/openshift-apiserver /usr/bin/
error: build error: error building at STEP "COPY --from=bu...t-apiserver/openshift-apiserver": no such file or directory
```

File: ci-operator/config/openshift/openshift-apiserver/openshift-openshift-apiserver-release-4.4.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ images:
   inputs:
     bin:
       as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
```
Comment (tnozicka) [inline ci-operator/config/openshift/openshift-apiserver/openshift-openshift-apiserver-release-4.4.yaml:20]: matching https://github.com/openshift/openshift-apiserver/blob/219e778/images/Dockerfile.rhel#L1

**Pull request merged**